CHANGES.md

```diff
@@ -0,0 +1,6303 @@
+# GAP - history of changes
+
+### Changes in GAP 4.11.0 (February 2020)
```

Comment:
The nesting levels here are wrong: this should be H2 not H3, i.e.:
```suggestion
## Changes in GAP 4.11.0 (February 2020)
```
And then everything afterwards pertaining to GAP 4.11 needs to be nested one level deeper...